.rubocop.yml

```diff
@@ -791,6 +791,7 @@ Style/PredicateName:
   # helpers in the form of `have_something` or `be_something`.
   Exclude:
     - '**/spec/**/*'
+    - '**/test/**/*'
```

Comment:
Yeah. The convention is usually using `spec` for RSpec suites, `test` for Minitest. Not following the convention bit us here, but it was a very small bite. Adding this exception should be enough.